gno.land/pkg/integration/testing_integration.go

```diff
@@ -10,15 +11,20 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/gnolang/gno/gno.land/pkg/gnoland"
 	"github.com/gnolang/gno/gnovm/pkg/gnoenv"
 	"github.com/gnolang/gno/tm2/pkg/bft/node"
 	"github.com/gnolang/gno/tm2/pkg/commands"
+	"github.com/gnolang/gno/tm2/pkg/crypto/bip39"
 	"github.com/gnolang/gno/tm2/pkg/crypto/keys"
 	"github.com/gnolang/gno/tm2/pkg/crypto/keys/client"
 	"github.com/gnolang/gno/tm2/pkg/log"
+	"github.com/gnolang/gno/tm2/pkg/std"
 	"github.com/rogpeppe/go-internal/testscript"
 )
 
+const numTestAccounts int = 4
```

Comment:
Why have this be a constant?

You only use it once when generating test accounts

Reply:
Why not have it as a constant?  It's a value that isn't meant to change and naming the value makes it clear what the value is meant to represent.

Reply:
I don't see a reason to expose implementation details like this (the number of initial accounts generated for the node) package-wide, especially since it isn't shared between functionality in any way (or exposed for that matter). 

Why extract it then? 

It's a nitpick for sure, I'm just trying to avoid having extracted functionality that isn't shared at all, or will probably ever change. 

I disagree with the need to even have initial test accounts for txtar node starts (which would mitigate this comment completely), so I've opened an issue on the matter where we can discuss, to avoid further blocking this PR:
https://github.com/gnolang/gno/issues/1484#issue-2053723528